doc/deployment-on-arbitrary-cluster.md

```diff
@@ -0,0 +1,30 @@
+# Deployment on Arbitrary Cluster
+
+In this guide we will cover installation of Multus and
+ovs-cni on your arbitrary cluster.
+
+# Requirements
+
+## A Cluster
+
+This guide requires you to have your own Kubernetes cluster. If you
+don't have one and just want to try Kubemacpool out, please refer to the [deployment
+on local cluster](deployment-on-local-cluster.md) guide.
+
+## Prerequisites
```

Comment:
What about OVS CNI?

Reply:
added. but why is it needed? I'm not sure I fully understand